topo_processor/cli/validate.py

```diff
@@ -3,54 +3,73 @@
 import click
 from linz_logger import LogLevel, get_log, set_level
 
-from topo_processor.file_system.get_fs import is_s3_path
+from topo_processor.stac.data_type import DataType
 from topo_processor.stac.validation import validate_stac
 from topo_processor.util import time_in_ms
+from topo_processor.util.configuration import lds_cache_local_tmp_folder
+from topo_processor.util.files import empty_dir
 
 
 @click.command()
+@click.option(
+    "-l",
+    "--layer",
+    required=False,
+    help="The layer number to validate. Take the last version by default.",
+)
+@click.option(
+    "-v",
+    "--version",
+    required=False,
+    help="(OPTIONAL) The specific version number to validate.",
+)
 @click.option(
```

Comment:
we should remove the source parameter from here too. 

We could override the source location possibly with a environment variable if we really want to run custom locations ideally we should just have a 

```
--metadata ./foo.csv
--metadata s3://foo/bar.csv
```

Reply:
OK, always the latest version by default then

Reply:
Yeah, lets make it super simple to run